cmd/kube-apiserver/app/aggregator.go

```diff
@@ -260,6 +260,7 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 	{Group: "batch", Version: "v1beta1"}:                         {group: 17400, version: 9},
 	{Group: "batch", Version: "v2alpha1"}:                        {group: 17400, version: 9},
 	{Group: "certificates.k8s.io", Version: "v1"}:                {group: 17300, version: 15},
```

Comment:
hmm, should there be an entry for `certificates.k8s.io/v1beta1`?

Reply:
not yet, once these types graduate to v1beta1 that'll be added